rapier/src/pipeline.rs

```diff
@@ -28,7 +28,6 @@ pub(crate) fn step(
     physics_controller: Res<'_, PhysicsTime>,
 ) {
     let gravity = Vec3::from(*gravity).into_rapier();
-    let mut integration_parameters = integration_parameters.to_owned();
     integration_parameters.dt *= physics_controller.scale;
```

Comment:
As you are modifying the same instance over and over, you would be multiplying the scale in each step call. That means that a scale of 0.5 would turn into a 0 dt after a while. Basically, "cloning" the element was to just mutate it locally and avoid this problem.

Reply:
Ah yes indeed. Thanks for pointing out.